package.json

```diff
@@ -9,6 +9,7 @@
     "bs58": "^4.0.1",
     "buffer": "^5.1.0",
     "firebase": "^4.12.0",
+    "ipfs": "^0.28.2",
```

Comment:
i've only used the Go ipfs implementation. Project page for js says "still in Alpha" but if it work, it works! :D